authentication/assuming-an-aws-iam-role-for-a-service-account.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="assuming-an-aws-iam-role-for-a-service-account"]
+= Assuming an AWS IAM role for a service account
+include::_attributes/common-attributes.adoc[]
+ifdef::openshift-rosa,openshift-dedicated[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+endif::openshift-rosa,openshift-dedicated[]
+:context: assuming-an-aws-iam-role-for-a-service-account
+
+toc::[]
+
+[role="_abstract"]
+ifdef::openshift-rosa[]
+{product-title} with AWS Security Token Service (STS) clusters include a pod identity webhook for use with pods running in user-defined projects.
+endif::openshift-rosa[]
+
+You can use the pod identity webhook to enable a service account to automatically assume an AWS Identity and Access Management (IAM) role in your own pods. If the assumed IAM role has the required AWS permissions, the pods can run AWS SDK operations for your AWS account by using temporary STS credentials.
```

Comment:
Personally I think "run AWS SDK operations for your AWS account" can be just changed to "run AWS SDK operations". There are several places with same issues in the chapter.

Reply:
Thank you, that's a good suggestion. I will remove "for your AWS account" throughout the SDK references in the PR.